mhkit/dolfyn/io/rdi.py

```diff
@@ -630,23 +780,53 @@ def read_cfgseg(self,):
         cfg['use_pitchroll'] = ['no', 'yes'][(coord_sys & 4) == 4]
         cfg['use_3beam'] = ['no', 'yes'][(coord_sys & 2) == 2]
         cfg['bin_mapping'] = ['no', 'yes'][(coord_sys & 1) == 1]
-        cfg['xducer_misalign_deg'] = fd.read_i16(1) * .01
+        cfg['heading_misalign_deg'] = fd.read_i16(1) * .01
         cfg['magnetic_var_deg'] = fd.read_i16(1) * .01
         cfg['sensors_src'] = np.binary_repr(fd.read_ui8(1), 8)
         cfg['sensors_avail'] = np.binary_repr(fd.read_ui8(1), 8)
-        cfg['bin1_dist_m'] = fd.read_ui16(1) * .01
-        cfg['xmit_pulse'] = fd.read_ui16(1) * .01
+        cfg['bin1_dist_m'] = round(fd.read_ui16(1) * .01, 4)
+        cfg['transmit_pulse_m'] = fd.read_ui16(1) * .01
         cfg['water_ref_cells'] = list(fd.read_ui8(2))  # list for attrs
-        cfg['fls_target_threshold'] = fd.read_ui8(1)
+        cfg['false_target_threshold'] = fd.read_ui8(1)
         fd.seek(1, 1)
-        cfg['xmit_lag_m'] = fd.read_ui16(1) * .01
+        cfg['transmit_lag_m'] = fd.read_ui16(1) * .01
         self._nbyte = 40
+
+        if cfg['prog_ver'] >= 8.14:
+            cpu_serialnum = fd.read_ui8(8)
+            self._nbyte += 8
+        if cfg['prog_ver'] >= 8.24:
+            cfg['bandwidth'] = fd.read_ui16(1)
+            self._nbyte += 2
+        if cfg['prog_ver'] >= 16.05:
+            cfg['power_level'] = fd.read_ui8(1)
+            self._nbyte += 1
+        if cfg['prog_ver'] >= 16.27:
+            # cfg['navigator_basefreqindex'] = fd.read_ui8(1)
+            fd.seek(1, 1)
+            cfg['serialnum'] = fd.read_ui32(1)
+            cfg['beam_angle'] = fd.read_ui8(1)
+            self._nbyte += 6
+        # # Documented in latest PDDecoder 10/12/22 but must be instrument specific
+        # if cfg['prog_ver'] >= 56:
+        #     fd.seek(1, 1)
+        #     pings_per_ensemble = fd.read_ui16(1)
+        #     exact_freq = fd.read_ui8(3)
+        #     #cfg['exact_freq'] = int("".join(str(x) for x in exact_freq))
+        #     self._nbyte += 6
+
```

Comment:
Looks like this was replaced by the definition above. Do we want to keep this?

Reply:
I'll remove it for now from the main branch until it's needed. I'm not entirely sure why it's in their documentation.